### ISSUE TITLE
AzurePipelines: Package prometheus-net.AspNetCore 6.0.0 is not compatible with net60

### ISSUE THREAD

**Issue by SquallHalle** [prometheus-net/prometheus-net#363]
Hey,

i still get this build error:

`[error]The nuget command failed with exit code(1) and error(NU1202: Package prometheus-net.AspNetCore 6.0.0 is not compatible with net60 (.NETFramework,Version=v6.0). Package prometheus-net.AspNetCore 6.0.0 supports: netcoreapp3.1 (.NETCoreApp,Version=v3.1)`

**Comment by andreyshiryaev** [comment]
I have the same issue

![image](https://user-images.githubusercontent.com/14107563/183098385-01086f8b-05af-4d66-81ab-0ca17b261d7a.png)

**Comment by sandersaares** [comment]
How can I reproduce this error?

**Comment by andreyshiryaev** [comment]
> How can I reproduce this error?


you should have 
1. The Project with NET6
2. Add this nugget
3. The error is when building a solution in release mode in the Azure Pipeline

**Comment by walwe** [comment]
I solved this by upgrading nuget to >=6.0:
```
- task: NuGetToolInstaller@1
  inputs:
    versionSpec: 6.0
```

**Comment by sandersaares** [comment]
It sounds like this may be related to an old NuGet version? There are many .NET 6 projects using this library, so it seems unlikely that the library package is broken. However, if upgrading NuGet version does not help, please post a new issue with more details (e.g. example pipeline).
